bin/run_ap_pipe.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Set up the stack and necessary ap_packages
+source /software/lsstsw/stack/loadLSST.bash
+setup lsst_distrib
+
+# Set up the cloned and pre-sconsed ap_packages
+L1DBPROTO_DIR=/project/mrawls/l1dbproto
```

Comment:
Suggest for all of these that you either allow them to be passed as arguments or use `os.getenv` to get environment variables.  I just think we'll quickly find it convenient to be able to run these scripts without editing them.